src/js/controllers/ctrl-media-import.js

```diff
@@ -1,9 +1,10 @@
 const app = angular.module('materia')
 
-app.controller('MediaImportCtrl', function ($scope, $window, $timeout) {
+app.controller('MediaImportCtrl', function ($scope, $window, $timeout, AssetSrv) {
 	const SORTING_NONE = false
 	const SORTING_ASC = 'asc'
 	const SORTING_DESC = 'desc'
+	let IS_HIDDEN_CLICK = false
```

Comment:
Typically all-caps variable names are used for static constants, as seen above. This is a variable, so it should follow the camel case pattern of `isHiddenClick`.